app/hotels/src/filter/stars/__tests__/StarsCheckbox.test.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+import * as React from 'react';
+import ShallowRenderer from 'react-test-renderer/shallow';
+
+import StarsCheckbox from '../StarsCheckbox';
+
+const renderer = new ShallowRenderer();
+
+describe('StarsCheckbox', () => {
+  it('render unchecked with 1 star and 1 hotel', () => {
```

Comment:
I honestly hate these tests. I think the value is zero here (not literally - you are conservating it in working state, but that's it). I have no idea whether for example this is ok or not:

```
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#757575",
        "fontSize": 12,
        "marginLeft": 3,
      }
    }
  >
    123
     hotel
    s
  </Text>
```

🙂 We will hopefully improve this soon. I would like to introduce stupid simple playground where you can see this test results. Kinda like Storybook but without all the pain around stories...